projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -5306,5 +5306,468 @@ public string RegistrationDate
                 }
             }
         }
+
+        /// <summary>Certifier name.</summary>
+        /// <value>the certifier's name</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.CertifierName = "Janet Seito";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Certifier's Name: {ExampleBirthRecord.CertifierName}");</para>
+        /// </example>
+        [Property("Certifier Name", Property.Types.String, "Birth Certification", "Name of Certifier.", true, VR.IGURL.Practitioner, true, 6)]
+        [FHIRPath("Bundle.entry.resource.where($this is Practitioner).where(extension.value ='certifier')", "name")]
+        public string CertifierName
+        {
+            get
+            {
+                if (Certifier != null && Certifier.Name != null)
+                {
+                    return Certifier.Name.FirstOrDefault()?.Text;
+                }
+                return null;
+            }
+            set
+            {
+                if (Certifier == null)
+                {
+                    CreateCertifier();
+                }
+                HumanName name = Certifier.Name.FirstOrDefault();
+                if (name != null && !String.IsNullOrEmpty(value))
+                {
+                    name.Text = value;
+                }
+                else if (!String.IsNullOrEmpty(value))
+                {
+                    name = new HumanName();
+                    name.Use = HumanName.NameUse.Official;
+                    name.Text = value;
+                    Certifier.Name.Add(name);
+                }
+            }
+        }
+
+        /// <summary>Certifier NPI</summary>
+        /// <value>the certifiers npi</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.CertifierNPI = "123456789011";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Certifier NPI: {ExampleBirthRecord.CertifierNPI}");</para>
+        /// </example>
+        [Property("Certifier NPI", Property.Types.String, "Birth Certification", "Certifier's NPI.", true, VR.IGURL.Practitioner, true, 13)]
+        [FHIRPath("Bundle.entry.resource.where($this is Practitioner).where(extension.value ='certifier').identifier.where(system='http://hl7.org/fhir/sid/us-npi')", "value")]
+        public string CertifierNPI
+        {
+            get
+            {
+                return Certifier?.Identifier?.Find(id => id.System == "http://hl7.org/fhir/sid/us-npi")?.Value;
```

Comment:
This is the value for CodeSystems.US_NPI_HL7 ? Can you use that variable here instead